org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/annotations/services/XbaseWithAnnotationsGrammarAccess.java

```diff
@@ -54,15 +54,17 @@
 		
 		//XAnnotation:
 		//	{XAnnotation}
-		//	'@' annotationType=[types::JvmAnnotationType|QualifiedName] (=> '(' (elementValuePairs+=XAnnotationElementValuePair
-		//	(',' elementValuePairs+=XAnnotationElementValuePair)*
+		//	'@' annotationType=[types::JvmAnnotationType|QualifiedName] (=>'(' (elementValuePairs+=XAnnotationElementValuePair (
```

Comment:
Unexpected line-break after the opening parenthesis if there is no line-break before the closing parenthesis and no indentation applied. To be discussed if we want to do something about this and if so, what we want to do about this.

Reply:
With latest patch the following
```
XAnnotation :
	{XAnnotation}
	'@' annotationType=[types::JvmAnnotationType | QualifiedName] (=>'('
		(
			elementValuePairs+=XAnnotationElementValuePair (',' elementValuePairs+=XAnnotationElementValuePair)*
		|	value=XAnnotationElementValueOrCommaList
		)?
	')')?
;
```
will result in
```
XAnnotation:
	{XAnnotation}
	'@' annotationType=[types::JvmAnnotationType|QualifiedName] (=>'('
	(elementValuePairs+=XAnnotationElementValuePair (',' elementValuePairs+=XAnnotationElementValuePair)*
	| value=XAnnotationElementValueOrCommaList)?
	')')?;
```
Looks ok for me.